docs/2.9.0/docs/canton/usermanual/FAQ.rst

```diff
@@ -163,6 +163,17 @@ system:
 
 .. literalinclude:: /canton/includes/mirrored/community/app/src/test/resources/documentation-snippets/dev-version-support.conf
 
+How do I enable preview features?
+---------------------------------
+For early access to new features, enable preview version support. A preview protocol version allows the use of features with initial availability.
+Note that we offer support for IA features only under specific agreements. Customers using Daml Enterprise can receive support for IA features
```

Comment:
```suggestion
Note that we offer support for IA features only under explicit agreements. Customers using Daml Enterprise can receive support for IA features
```
I think explicit is the right word here, though admittably it's repeated in the next sentence.